x-pack/legacy/plugins/uptime/public/components/functional/filter_group/filter_group.tsx

```diff
@@ -21,6 +21,7 @@ interface FilterBarQueryResult {
 interface FilterBarDropdownsProps {
   currentFilter: string;
   onFilterUpdate: (kuery: string) => void;
+  setFilters: Function;
```

Comment:
```suggestion
  setFilters: (filters: Map<string, string[]>) => any;
```

We can provide a more specific description of what this function does.